### PR TITLE
GO-REST: Header access

### PR DESCRIPTION
**Pull request by jeffidol-wf**
# Providing Response Header Access for REST endpoint

## Code Review
* https://github.com/jeffidol-wf/go-rest/pull/1

### CODE REVIEW

**Comment by tylertreat-wf** [comment]
@alexandercampbell-wf @aaronkavlie-wf @stevenosborne-wf

**Comment by tylertreat-wf** [comment]
+1

**Comment by jeffidol-wf** [comment]
FYI: @mattsanders-wf @jacobcabantomski-wf

File: rest/rest.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 Workiva, LLC
+Copyright 2014 - 2015 Workiva, LLC
```
Comment (alexandercampbell-wf) [inline rest/rest.go]: We should make a PR to update all these headers at some point.

**Comment by alexandercampbell-wf** [comment]
+1

File: rest/rest.go (hunk 1 above)
Comment (jeffidol-wf) [inline rest/rest.go]: On it

Comment (alexandercampbell-wf) [inline rest/rest.go]: Unrelated to the header stuff obviously.

Comment (jeffidol-wf) [inline rest/rest.go]: https://github.com/Workiva/go-rest/pull/55

**Comment by stevenosborne-wf** [comment]
+1

**Comment by teresarevious-wf** [comment]
@rmconsole-wf

**Pull request merged**